### PR TITLE
fix(pr252): address comments for real this time

### PR DESCRIPTION
**Pull request by sdwoodbury**
### What this PR does 📖

- what the title says. see #252 to see the suggestions. 

### Which issue(s) this PR fixes 🔨

- Resolve #

### CODE REVIEW

**Comment by sdwoodbury** [comment]
> warning The title of this PR is invalid. Please make the title match the regex `(?:add|update|task|chore|feat|fix)\([a-z-A-Z]+\):\s.+`. e.g.) `add(cli): enable --verbose flag`, `fix(api): avoid unexpected error in handler`

numbers should be allowed in the title

**Review by phillsatellite — DISMISSED** [review]

**Comment by phillsatellite** [comment]
Looked good to me, added Do not Merge because PR needs reviewed by a dev

File: kit/src/elements/file/mod.rs [hunk 1]
```diff
@@ -28,10 +27,14 @@ pub struct Props<'a> {
 
 pub fn get_text(file_name: String) -> (String, String) {
     let mut file_name_formatted = file_name.clone();
-    let file_extension = std::path::Path::new(&file_name)
+    // don't append a '.' to a file name if it has no extension
+    let file_extension = match std::path::Path::new(&file_name)
         .extension()
         .and_then(OsStr::to_str)
-        .unwrap_or_default();
```
Comment (dariusc93) [inline kit/src/elements/file/mod.rs:34]: could use a map and do eg

```rust
let file_extension = PathBuf::from(&file_name)
          .extension()
          .and_then(OsStr::to_str)
          .map(|s| format!(".{s}"))
          .unwrap_or_default();
```

**Review by dariusc93 — DISMISSED** [review]
LGTM. Did leave a small suggestion but its non-blocking to this PR

**Review by lgmarchi — DISMISSED** [review]
LGTM. Just need to solve conflicts.

**Comment by sdwoodbury** [comment]
changes complete. need someone to "approve" the review before it gets merged

**Review by lgmarchi — APPROVED** [review]

**Pull request merged**